### ISSUE TITLE
ERROR: "xauth:  unable to rename authority file /tmp/.docker.xauth, use /tmp/.docker.xauth-n"

### ISSUE THREAD

**Issue by caozx1110** [larics/icuas22_competition#19]
Firstly, I have run the docker container successfully(--focal-nogpu).
Tonight, when I up the container, the ubuntu system report an unknown error and the Gazebo UI didn't show up.
Then I rerun the container, use the `./docker_run.sh --focal-nogpu` order, but it report an error:

> xauth:  unable to rename authority file /tmp/.docker.xauth, use /tmp/.docker.xauth-n

and again the Gazebo UI disappear.
I cannot find out the solution, so may I ask for your help?
Thanks a lot !

**Comment by lmark1** [comment]
Hi @caozx1110,

Just to confirm, were you able to successfully run the simulation using Docker at some point ?

How did you start the container the second time? Generally you want to use the ```./docker_run.sh``` once to create the container and use the following commands when you want to start again:
```bash
docker start icuas22_competition_[[DISTRO]]
docker attach icuas22_competition_[[DISTRO]]
```
where [[DISTRO]] can be either bionic, focal or focal-nogpu.

If this does not solve your issue please try removing the container, rebuilding the image and rerunning the container. 

If this does not solve your issue please post terminal screenshots.

**Comment by caozx1110** [comment]
Hi @lmark1 
It is a weird thing when I turn on the PC today, it can work successfully.

For this:

> How did you start the container the second time? Generally you want to use the ./docker_run.sh only once to create the container and use the following commands when you want to start it again:

In fact, I used the `docker start` and `docker exec` to start the existing container.

Yesterday I have tried to rerun container and rerun, even rebuild the image.
And I have tried reboot the PC, too !!!

![2022-02-24 23-55-46 的屏幕截图](https://user-images.githubusercontent.com/78025946/155644101-b677c47e-8515-46c0-b597-9d5e67ad90d6.png)

this is the screenshot with error.

I don't know how the temp file come back, lol. Anyway it can work now.
